templates/privacy.mustache

```diff
@@ -4,8 +4,14 @@
 		<!-- Meta -->
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<meta http-equiv="Content-Type" content="text/html;charset=utf-8">
-		<title>Manzo's Marbles</title>
+		<title>Privacy - Manzo's Marbles</title>
 		<link href="favicon.ico" rel="shortcut icon" type="image/x-icon">
+		<meta name="theme-color" content="#ba0069">
+		{{>partials/metaBasics}}
+		<meta property="og:title" content="Privacy - Manzo's Marbles" />
+		<meta property="og:description" content="A tool for web-marbles that allows for building custom levels" />
```

Comment:
Should this be the description of the privacy page? It's not on the Terms and Conditions page in any case.